python/ray/data/read_api.py

```diff
@@ -135,12 +135,12 @@ def range(n: int, *, parallelism: int = 200) -> Dataset[int]:
 
 
 @PublicAPI
-def range_arrow(n: int, *, parallelism: int = 200) -> Dataset[ArrowRow]:
-    """Create an Arrow dataset from a range of integers [0..n).
+def range_table(n: int, *, parallelism: int = 200) -> Dataset[ArrowRow]:
```

Comment:
Could we add a deprecation placeholder?

```
def range_arrow(*a, **kw) -> None:
   raise DeprecationWarning("Use range_table() instead of range_arrow()")
```

And the appropriate imports.